core/src/main/scala/za/co/absa/spline/harvester/plugin/embedded/DeltaPlugin.scala

```diff
@@ -75,7 +75,7 @@ class DeltaPlugin(
       val syntheticSourceRead = SyntheticDeltaRead(sourceRelation.output, sourceId, Map.empty[String, Any], sourceRelation)
       val params = Map.empty[String, Any]
 
-      val condition = extractValue[Expression](command, "condition").toString
+      val condition = extractValue[Option[Expression]](command, "condition").toString
```

Comment:
Is `Option.toString` intended here? Shouldn't it be `.map(_.toString)`?

Reply:
In DELETE and UPDATE case the params are set directly so I wanted to avoid going through OperationParamsConverter in this case only.